usaspending_api/recipient/management/sql/restock_recipient_profile.sql

```diff
@@ -20,7 +20,14 @@ CREATE MATERIALIZED VIEW public.temporary_recipients_from_transactions_view AS (
     ))::uuid AS recipient_hash,
     COALESCE(fpds.awardee_or_recipient_uniqu, fabs.awardee_or_recipient_uniqu) AS recipient_unique_id,
     COALESCE(fpds.ultimate_parent_unique_ide, fabs.ultimate_parent_unique_ide) AS parent_recipient_unique_id,
+    MD5(UPPER(
+      CASE
+        WHEN COALESCE(fpds.ultimate_parent_uei, fabs.ultimate_parent_uei) IS NOT NULL THEN CONCAT('uei-', COALESCE(fpds.ultimate_parent_uei, fabs.ultimate_parent_uei))
+        WHEN COALESCE(fpds.ultimate_parent_unique_ide, fabs.ultimate_parent_unique_ide) IS NOT NULL THEN CONCAT('duns-', COALESCE(fpds.ultimate_parent_unique_ide, fabs.ultimate_parent_unique_ide))
+        ELSE CONCAT('name-', COALESCE(fpds.ultimate_parent_legal_enti, fabs.ultimate_parent_legal_enti, '')) END
```

Comment:
Do we use this pattern elsewhere for deriving a hash from transactions ... where, if uei, duns, AND name are all null, the hash falls back to just `MD5(UPPER('name-'))`?
(I guess we do it directly above for `recipient_hash` ...)

We can continue for consistency if so. But I guess it means that a lot of children have the potential to be bundled under a no-name (blank string) parent recipient. Wondering if a null hash (if possible) would be better there.

Reply:
Or, does it not matter since this is a temp-use matview that gets thrown away?

Reply:
We're getting rid of the parent_recipient_hash since we are no longer using the hashes for `recipient_affiliation`